src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt

```diff
@@ -242,10 +245,18 @@ class ResettlementAssessmentService(
         )
       },
     )
-
+    auditReportSubmission(assessmentType, nomsId, name)
     return ResettlementAssessmentSubmitResponse(deliusCaseNoteFailed = failedCaseNotes.isNotEmpty())
   }
 
+  private fun auditReportSubmission(assessmentType: ResettlementAssessmentType, nomsId: String, userName: String) {
+    val auditAction = when (assessmentType) {
+      ResettlementAssessmentType.BCST2 -> AuditAction.IMMEDIATE_NEEDS_REPORT_SUBMITTED_SUCCESS
```

Comment:
There's probably a few other things we could audit as well such as completing a report for a pathway (as this is write information to the database) and viewing the report information.